emcc.py

```diff
@@ -187,6 +187,10 @@ def base64_encode(b):
   return b64.decode('ascii')
 
 
+def round_to(number, size):
```

Comment:
How about `round_up` as the rounding here is upwards and not downwards?

Or, how about `align_up` which would match `align_memory` in `emscripten.py` which does the same but for the 16-byte memory alignment?

(another option is since all this file aligns to is the wasm page size, so `align_wasm_page_size`)